bin/oneoff/pullOutBlocks.js

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/node
+
+/*
+  This was a one off script used to create .json files for each block in App Lab and Game Lab.
+  The json files will be used to seed ProgrammingExpressions.
+ */
+
+var fs = require("fs");
+
+var applabBlocksToImport = [
+  // Add dropletConfig definition of the blocks you want to import here
```

Comment:
Ya you have to do some work to remove things that are dynamically generated such as dropdowns or have constants. Right now those fields were not needed so I just ran the script and when it complained about a line for those reasons and I didn't need it I just deleted the line.

Reply:
If it would save you time, one idea would be to change our application code to print out the droplet config, at which point it would have all the constants already substituted.